code/modules/admin/verbs/gimmick_team.dm

```diff
@@ -52,7 +52,7 @@
 			players_to_spawn += candidate
 	else
 		to_chat(src, "Polling candidates...")
-		players_to_spawn = pollCandidates("Do you want to play as an event character?")
+		players_to_spawn = SSghost_spawns.poll_candidates("Do you want to play as an event character?")
```

Comment:
If you're changing this anyway, might be good to include the name of the outfit. Not required, just an idea.
E.g: Do you want to play as a Soviet Tourist?
Will end the "what type of event char?!?!" questions in deadchat.